server/server.ts

```diff
@@ -39,6 +39,14 @@ if (conf.DOMAIN === 'thegulocal.com') {
 
 server.use(helmet());
 
+server.use(function (_: Request, res: Response, next: NextFunction) {
+	res.setHeader(
+		'Content-Security-Policy-Report-Only',
+		'report-to /csp-audit-report-endpoint',
```

Comment:
the documentation is inconsistent about how report-to works https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to
it sounds like report-uri has recently been deprecated so it may not be completely up to date in the docs?

Reply:
I've updated it to send both report-uri and report-to ... which should give us the best browser coverage (report-uri is ignored for browsers that support report-to apparently, also firefox at the time of writing doesn't support report-to)